### ISSUE TITLE
The latest version of the library is different between NPM and CDN

### ISSUE THREAD

**Issue by woneob** [mixpanel/mixpanel-js#402]
The library version in the master branch on GitHub and the latest release version published on NPM is `2.47.0`. 
However, the CDN's library version is `2.48.0`.  

GitHib
https://raw.githubusercontent.com/mixpanel/mixpanel-js/master/dist/mixpanel.globals.js
```js
    var Config = {
        DEBUG: false,
        LIB_VERSION: '2.47.0'
    };
```

CDN: https://cdn.mxpnl.com/libs/mixpanel-2-latest.js
```js
    var Config = {
        DEBUG: false,
        LIB_VERSION: '2.48.0'
    };
```

Why is that different?
Could you please publish the latest version to the NPM registry?

**Comment by woneob** [comment]
It was released a few hours ago as v2.48.0. 👏

**Comment by tdumitrescu** [comment]
Hi @woneob, yes we often lead with the CDN for a bit before publishing to NPM. 2.48.0 is now out everywhere; release notes are here: https://github.com/mixpanel/mixpanel-js/releases/tag/v2.48.0
